zygrader/ui/components.py

```diff
@@ -696,6 +701,18 @@ def _sort(self, lines):
         lines.sort(key=lambda line: line.sort_index)
         return lines
 
+    def _draw_list_lines(self, window, lines, y_start, x_start):
+        for line_number, line in enumerate(lines):
+            prefix = ScrollableList.UNSELECTED_PREFIX
+            attributes = 0
+            if line_number + self._scroll == self._selected_index and not self._paged:
+                prefix = ScrollableList.SELECTED_PREFIX
+                attributes = curses.A_BOLD
```

Comment:
Just because that conditional is getting long, a function abstraction will help to keep it readable.
```suggestion
        def should_show_selected(line_number):
            is_selected = line_number + self._scroll == self._selected_index
            return is_selected and not self._paged

        for line_number, line in enumerate(lines):
            prefix = ScrollableList.UNSELECTED_PREFIX
            attributes = 0
            if should_show_selected(line_number):
                prefix = ScrollableList.SELECTED_PREFIX
                attributes = curses.A_BOLD
```